### PR TITLE
DAOS-15085 container: increase capa ULT stack size

### PR DESCRIPTION
**Pull request by wangdi1**
Increase capa ULT stack sizeto avoid stack overflow.

Required-githooks: true

### Before requesting gatekeeper:

* [ ] Two review approvals and any prior change requests have been resolved.
* [ ] Testing is complete and all tests passed or there is a reason documented in the PR why it should be force landed and forced-landing tag is set.
* [ ] `Features:` (or `Test-tag*`) commit pragma was used or there is a reason documented that there are no appropriate tags for this PR.
* [ ] Commit messages follows the guidelines outlined [here](https://daosio.atlassian.net/wiki/spaces/DC/pages/11133911069/Commit+Comments).
* [ ] Any tests skipped by the ticket being addressed have been run and passed in the PR.

### Gatekeeper:

* [ ] You are the appropriate gatekeeper to be landing the patch.
* [ ] The PR has 2 reviews by people familiar with the code, including appropriate owners.
* [ ] Githooks were used. If not, request that user install them and check copyright dates.
* [ ] Checkpatch issues are resolved.  Pay particular attention to ones that will show up on future PRs.
* [ ] All builds have passed.  Check non-required builds for any new compiler warnings.
* [ ] Sufficient testing is done. Check feature pragmas and test tags and that tests skipped for the ticket are run and now pass with the changes.
* [ ] If applicable, the PR has addressed any potential version compatibility issues.
* [ ] Check the target branch.   If it is master branch, should the PR go to a feature branch?  If it is a release branch, does it have merge approval in the JIRA ticket.
* [ ] Extra checks if forced landing is requested
  * [ ] Review comments are sufficiently resolved, particularly by prior reviewers that requested changes.
  * [ ] No new NLT or valgrind warnings.  Check the classic view.
  * [ ] Quick-build or Quick-functional is not used.
* [ ] Fix the commit message upon landing. Check the standard [here](https://daosio.atlassian.net/wiki/spaces/DC/pages/11133911069/Commit+Comments). Edit it to create a single commit. If necessary, ask submitter for a new summary.

### CODE REVIEW

**Review by liuxuezhao — APPROVED** [review]

**Comment by wangdi1** [comment]
failure due to DAOS-14353

File: src/container/container_iv.c [hunk 1]
```diff
@@ -1008,7 +1008,7 @@ cont_iv_hdl_fetch(uuid_t cont_hdl_uuid, uuid_t pool_uuid,
 	arg.eventual = eventual;
 	arg.invalidate_current = invalidate_current;
 	rc = dss_ult_create(cont_iv_capa_refresh_ult, &arg, DSS_XS_SYS,
-			    0, 0, NULL);
+			    0, DSS_DEEP_STACK_SZ, NULL);
```
Comment (NiuYawei) [inline src/container/container_iv.c:1011]: I think we'd investigate the IV operations deeper, and try to fix all the potential stack overrun issues together:

1. Do we know what are the major stack consumers for the IV operations? Are these functions from Cart or DAOS? If they are from Cart, the async IV operation in lazy mode should use deep stack size as well? See iv_op_async().
2. Should we use deep stack size for all container IV operations? See ds_cont_tgt_snapshots_refresh().
3. Should we use deep stack size for pool IV operations as well? See ds_pool_iv_svc_fetch().

**Review by NiuYawei — CHANGES_REQUESTED** [review]
Temporarily mark -1 until my concern is addressed.

File: src/container/container_iv.c (hunk 1 above)
Comment (NiuYawei) [reply src/container/container_iv.c:1011]: I'm fine with landing it as-is if it's blocking 2.4.2. Let's address my concerns in master.

**Review by NiuYawei — APPROVED** [review]

**Pull request merged**